office-addin/src/taskpane/helpers/WordHelper.ts

```diff
@@ -9,13 +9,19 @@ import { MindMapUUID } from "./MindMapGenHelper";
 
 export class WordHelper {
   static async getDocumentTitle(): Promise<string> {
+    const documentUrl = Office.context.document.url;
+    const filename = documentUrl.substring(
+      Math.max(documentUrl.lastIndexOf("\\"), documentUrl.lastIndexOf("/")) + 1,
+      documentUrl.lastIndexOf(".")
```

Comment:
What does the URL look like? Why using "." here as end?

Reply:
Trim off the file extension, so that "path/to/word title.docx" would be "word title".